roles/test_operator/tasks/main.yml

```diff
@@ -118,6 +118,50 @@
           )
         }}
 
+    - name: Delete CRDs created by test-operator
+      kubernetes.core.k8s:
+        kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+        api_key: "{{ cifmw_openshift_token | default(omit)}}"
+        context: "{{ cifmw_openshift_context | default(omit)}}"
+        kind: CustomResourceDefinition
+        state: absent
+        api_version: v1
+        name: "{{ item }}"
+        namespace: "{{ cifmw_test_operator_namespace }}"
+        wait: true
+        wait_timeout: 600
+      loop:
+        - tempests.test.openstack.org
+        - tobikoes.test.openstack.org
+        - ansibletests.test.openstack.org
+        - horizontests.test.openstack.org
+
+    - name: Install required packages
+      become: true
+      ansible.builtin.package:
+        state: present
+        name:
+          - git
+          - make
+
+    - name: Clone test-operator repository and checkout into specified version
+      ansible.builtin.git:
+        repo: "https://github.com/openstack-k8s-operators/test-operator.git"
+        dest: /tmp/test-operator
+        refspec: '+refs/pull/*:refs/heads/*'
+        version: "{{ cifmw_test_operator_version }}"
+        force: true
+
+    - name: Run make generate manifests install
```

Comment:
nit: suggest to add execution path for the title.
E.g.: `- name: Run make generate manifests install from /tmp/test-operator dir`